src/panels/config/automation/action/ha-automation-action-row.ts

```diff
@@ -251,7 +251,7 @@ export default class HaAutomationActionRow extends LitElement {
   }
 
   private _handleUiModeNotAvailable(ev: CustomEvent) {
-    this._warnings = handleStructError(ev.detail);
+    this._warnings = handleStructError(this.hass, ev.detail).warnings;
```

Comment:
The automation editor is using helpers of Lovelace, that is not correct, if it is a global helper move it out of Lovelace